src/sentry/static/sentry/app/types/index.tsx

```diff
@@ -997,19 +997,24 @@ export type NewQuery = {
   id: string | undefined;
   version: SavedQueryVersions;
   name: string;
-  projects: Readonly<number[]>;
+  createdBy?: User;
+
+  // Query and Table
+  query: string;
   fields: Readonly<string[]>;
   widths?: Readonly<string[]>;
-  query: string;
   orderby?: string;
+
+  // GlobalSelectionHeader
+  projects: Readonly<number[]>;
+  environment?: Readonly<string[]>;
   range?: string;
   start?: string;
   end?: string;
-  environment?: Readonly<string[]>;
-  tags?: Readonly<string[]>;
```

Comment:
Is there a reason why `tags?` was deleted here?

Reply:
We used to support event views with an explicit list of tags to show in the tag summary sidebar. We removed that feature and replaced it with automatic tag list generation. This is a remnant of the old behavior.